test_vm/src/expects.rs

```diff
@@ -43,7 +44,11 @@ impl Expect {
         sector_expiry: ChainEpoch,
     ) -> ExpectInvocation {
         let params = IpldBlock::serialize_cbor(&BatchActivateDealsParams {
-            sectors: vec![ActivateDealsParams { deal_ids: deals, sector_expiry }],
+            sectors: vec![ActivateDealsParams {
+                deal_ids: deals,
+                sector_expiry,
+                sector_type: RegisteredSealProof::StackedDRG8MiBV1,
```

Comment:
Please make the sector type a parameter. This will add a little noise, but necessary when it becomes optional to distinguish whether it was provided or not.